src/contracts/core/DelegationManager.sol

```diff
@@ -106,9 +119,11 @@ contract DelegationManager is
      */
     function registerAsOperator(
         OperatorDetails calldata registeringOperatorDetails,
+        uint32 allocationDelay,
         string calldata metadataURI
     ) external {
-        require(!isDelegated(msg.sender), "DelegationManager.registerAsOperator: caller is already actively delegated");
+        require(!isDelegated(msg.sender), ActivelyDelegated());
+        allocationManager.setAllocationDelay(msg.sender, allocationDelay);
```

Comment:
is this only called when initially registering as an operator for the moment?
do we have a transition plan for existing operators?

Reply:
nevermind, I see that there are other codepaths for adjusting.